helm/prefect-server/values.yaml

```diff
@@ -0,0 +1,273 @@
+labels: {}
+
+# matchLabels defines extra selection labels to merge into all
+# object configurations
+matchLabels: {}
+
+# annotations to merge into all object configurations
+# NOTE: These will not apply to the postgresql subchart
+# and must be defined within `postgresql` as well if that
+# is desired
+annotations: null
+
+# fqdnSuffix determines the string appended to names when
+# determining the fully qualified domain name of services for
+# internal communication
+fqdnSuffix: "svc.cluster.local"
+
+# postgresql configures the backing database
+# NOTE: the variables names must remain as is to match the
+# bitnami subchart even if the bitnami subchart is not being
+# used, these variables are used to create the connection
+# string
+postgresql:
+  postgresqlDatabase: prefect
+  postgresqlUsername: prefect
+
+  # existingSecret configures which secret should be referenced
+  # for access to the database. If null and `useSubChart` is
+  # enabled, the secret will be generated. If using an external
+  # postgres service, this value should be set to a secret
+  # containing the key `postgresql-password`
+  existingSecret: null
+
+  # postgresqlPassword sets the password to be used if
+  # `existingSecret` is not set. This is the password for
+  # `postgresqlUsername` and will be set within the secret at
+  # the key `postgresql-password`
+  # postgresqlPassword: use-a-strong-password
+
+  # servicePort configures the port that the database should be
+  # accessed at
+  servicePort: 5432
+
+  # externalHostname defines the address to contact an externally
+  # managed postgres database instance at. This is not required if
+  # `internalPostgres` is `true`
+  externalHostname: ""
+
+  # useSubChart determines if a this chart should deploy a
+  # user-manager postgres database or use an externally managed
+  # postgres instance. If `useSubChart` is `true`, the
+  # bitnami/postgresql subchart will be deployed
+  useSubChart: true
+
+  # postgresql configuration below here is only used if using
+  # the subchart
+
+  # persistence enables a PVC that stores the database between
+  # deployments. If making changes to the database deployment, this
+  # PVC will need to be deleted for database changes to take effect.
+  # This is especially notable when the authentication password
+  # changes on redeploys.
+  # This is disabled by default because we do not recommend using
+  # the subchart deployment for production deployments.
+  persistence:
+    enabled: false
+    size: 8Gi
+
+  initdbUser: postgres
+
+  initdbScripts:
+    create_pgcrypto.sql: |
+      -- create pgcrypto extension, required for Hasura UUID
+      CREATE EXTENSION IF NOT EXISTS pgcrypto;
+      CREATE EXTENSION IF NOT EXISTS "pg_trgm";
+      SET TIME ZONE 'UTC';
+
+# prefectConfig allows override of prefect_server/config.toml values
+# Values placed here will result in environment variable definitions
+# starting with "PREFECT_SERVER__" and will be passed to TODO
+prefectConfig:
```

Comment:
Instead of hardcoding some values, I'd suggest we leave this blank and put anything that is required into the images (I think this is already done). If this is more to document what a user might want to change, we can always handle that in docs or in a comment.

The issue with hardcoding values here is that if we change them in the code (update the `config.toml` in prefect-server) we'd also need to change them here to have them take effect.

Reply:
I'll try removing these, they were in the original PR and I agree they shouldn't be hardcoded if they match the current config.